src/test/java/org/jenkinsci/plugins/gitclient/CliGitAPIImplTest.java

```diff
@@ -27,7 +27,6 @@ protected boolean hasWorkingGetRemoteSymbolicReferences() {
     private void setCliGitDefaults() throws Exception {
         if (!cliGitDefaultsSet) {
             CliGitCommand gitCmd = new CliGitCommand(null);
-            gitCmd.setDefaults();
```

Comment:
What then is the purpose of the whole `setCliGitDefaults` method?

Reply:
I believe that it is trying to set the global defaults required of the git installation (must have a user defined and an email address defined otherwise it fails commits with a message that user and email address must be defined).  If I remember correctly, it is trying to set them only once per JVM to reduce failures where multiple JVM's running tests concurrently (`-forkCount=1C`) will collide with each other and cause failures.  However, that's just from memory, not a high precision idea.  I'll need to look at the git history of the file and the comments on the commits before I'm confident of the reasons.  I'm on vacation with my family currently and won't be back until the end of this week.  I can look at it then or other maintainers can investigate if it is needed sooner.

Reply:
> it is trying to set the global defaults required of the git installation

That is what this method _was_ doing, until this PR deleted the line that actually did something (AFAICT). Thus in #920 I just delete the now-useless method.